core-build/gulp-core-build/src/tasks/JestTask.ts

```diff
@@ -47,6 +47,16 @@ export interface IJestConfig {
    * Same as Jest CLI option testPathIgnorePatterns
    */
   testPathIgnorePatterns?: string[];
+
+  /**
+   * Same as Jest CLI option moduleDirectories
```

Comment:
>moduleDirectories [](start = 29, length = 17)

how does this affect the jest config? "What does this do?"

Reply:
Should be added to tasks to document all jest-related stuff here.